heka/meta/heka.yml

```diff
@@ -425,3 +427,88 @@ aggregator:
       max_file_size: 524288
       full_action: drop
 {%- endif %}
+os_telemetry_collector:
+  decoder:
+    sample:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/decoders/metering.lua
+      module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
+      config:
+        decoder: 'ceilometer'
+      {%- if os_telemetry_collector.resource_decoding is defined %}
+        decode_resources: true
+      {%- endif %}
+        metadata_fields: "status deleted container_format min_ram updated_at min_disk is_public size checksum created_at disk_format protected instance_host host display_name instance_id instance_type status state user_metadata.stack"
+  input:
+{%- if os_telemetry_collector.rabbit_host is defined %}
+    openstack_sample_amqp:
+      engine: amqp
+      user: {{ os_telemetry_collector.rabbit_user }}
+      password: {{ os_telemetry_collector.rabbit_password }}
+      port: {{ os_telemetry_collector.rabbit_port }}
+      host: {{ os_telemetry_collector.rabbit_host }}
+      vhost: {{ os_telemetry_collector.rabbit_vhost }}
+      queue: {{ os_telemetry_collector.rabbit_queue }}
+      routing_key: {{ os_telemetry_collector.rabbit_queue }}
+      decoder: sample_decoder
+      splitter: NullSplitter
+      exchange: "ceilometer"
+      exchange_type: "topic"
+      exchange_auto_delete: false
+      queue_auto_delete: false
+{%- endif %}
+  filter:
+{%- if os_telemetry_collector.influxdb_host is defined %}
+    ceilometer_influxdb_accumulator:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/filters/influxdb_accumulator.lua
+      module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
+      preserve_data: false
+      message_matcher: "Fields[aggregator] == NIL && Type =~ /ceilometer_samples$/"
+      ticker_interval: 1
+      config:
+        time_precision: "ns"
+        payload_name: 'sample_data'
+        flush_count: 500
+        bulk_metric_type_matcher: 'ceilometer_samples'
+{%- endif %}
+{%- if os_telemetry_collector.influxdb_host is defined %}
+  encoder:
```

Comment:
you'll get a wrong syntax influxdb_host is not defined but ES is defined:
```encoder:```  key will be missing